examples/mysql_with_identity_monad.ml

```diff
@@ -91,5 +106,7 @@ let () =
     get_user dbh ~id:2l >>= fun user -> print_user user; Mysql.disconnect dbh; Ok ()
   in
   match result with
-  | Ok () -> print_endline "All went well!"
-  | Error _ -> print_endline "An error occurred!"
+  | Ok () ->
+      print_endline "All went well!"
```

Comment:
It's a shame that each case can't be one line. Still prefer this to multiple cases on the same line

Reply:
Guess ocamlformat opts for

> foolish consistency is the hobgoblin of the little minds